enclave.go

```diff
@@ -67,14 +67,49 @@ type Enclave struct {
 
 // Config represents the configuration of our enclave service.
 type Config struct {
+	// SOCKSProxy must be set if
+	//   1) your enclave application should obtain a Let's Encrypt-signed
+	//      certificate (i.e., if UseACME is set to true)
+	// or if
+	//   2) your enclave application makes HTTP requests over the Internet.
+	// If so, set SOCKSProxy to "socks5://127.0.0.1:1080".
```

Comment:
Maybe mention this port number must match the one given to viproxy on the host instance?

Reply:
It actually doesn't have to because this variable only controls the in-enclave IP-to-VSOCK proxy. There's a viproxy instance running on both the host and inside the enclave. We should probably remove this variable from the config struct because it's confusing and there aren't many good reasons to change it.